create/how_to/digitalocean_droplet.md

```diff
@@ -11,69 +11,69 @@
 A "droplet" is a set of resources, as a Virtual Machine, or a Server, in which you can run your own applications.
 In any DigitalOcean page, when you are logged in, you will find a menu in the upper-right corner. Click on "Create" -> "Droplets".
 
-![Create droplet](/digitalocean/01.create.png)
+![Selecting Droplets Create cloud servers](/digitalocean/01.create.png)
```

Comment:
Perhaps we should be a bit more descriptive with the alt text throughout this guide? The user might be able to guess, but in most cases we don't say what the image actually is (a detail screenshot of the DO interface highlighting this or that functionality).

Reply:
I agree 💯
Unfortunately, the screenshots on this article are all cropped. 
Not sure what else can be said about this one: 
<img width="415" alt="Screen Shot 2022-01-17 at 11 33 33" src="https://user-images.githubusercontent.com/90181761/149719358-b6cd89f6-03e6-441e-ac7c-9e509bfcb10c.png">

Reply:
I think mentioning it is a detail of DO's interface is a good start, no?